CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## Unreleased
+
+### Added
+* **Breaking** Added additional field for `MeetingSessionCredentials`
```

Comment:
The change in `DefaultAudioClientObserver ` is also breaking, right?

Reply:
My understanding is we don't mark breaking for files in internal folder.